server/controllers/addToCart.js

```diff
@@ -0,0 +1,8 @@
+const { addToCart } = require('../database/queries');
+
+module.exports = (req, res) => {
+    const { quantity, product_id } = req.body;
+    const { userId } = req
```

Comment:
you need to get userID from request cookies which are a **JWT**, So you need to verify your JWT at the first place.
I suggest if you can handle this by a generic middleware, so you can use whenever you need to verify user